vcd/resource_vcd_lb_server_pool.go

```diff
@@ -0,0 +1,346 @@
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "Server pool description",
+			},
+			"algorithm": &schema.Schema{
+				Type:         schema.TypeString,
+				Required:     true,
+				Description:  "Balancing method for the service. One of ip-hash, round-robin, uri, leastconn, url, or httpheader",
+				ValidateFunc: validateCase("lower"),
+			},
+			"algorithm_parameters": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "Additional options for load balancing algorithm for http-header or url algorithms",
+			},
+			"monitor_id": &schema.Schema{
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "Load Balancer Service Monitor ID",
+			},
+			"enable_transparency": &schema.Schema{
+				Type:        schema.TypeBool,
+				Default:     false,
+				Optional:    true,
+				Description: "Makes client IP addresses visible to the backend servers",
+			},
+
+			"member": {
+				Optional: true,
+				ForceNew: false,
+				Type:     schema.TypeList,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"id": {
+							ForceNew:    false,
+							Computed:    true,
+							Type:        schema.TypeString,
+							Description: "Pool member id",
+						},
+						"condition": &schema.Schema{
+							Type:         schema.TypeString,
+							ForceNew:     false,
+							Required:     true,
+							ValidateFunc: validateCase("lower"),
+							Description:  "Defines member state. One of enabled, drain, disabled.",
+						},
+						"name": {
+							Required:    true,
+							ForceNew:    false,
+							Type:        schema.TypeString,
+							Description: "Name of pool member",
+						},
+						"ip_address": {
+							Optional:    true,
+							ForceNew:    false,
+							Type:        schema.TypeString,
+							Description: "IP address of member in server pool",
+						},
+						"port": {
+							Required:    true,
+							ForceNew:    false,
+							Type:        schema.TypeInt,
+							Description: "Port at which the member is to receive traffic from the load balancer",
+						},
+						"monitor_port": {
+							Required:    true,
+							ForceNew:    false,
+							Type:        schema.TypeInt,
+							Description: "Port at which the member is to receive health monitor requests",
+						},
+						"weight": {
+							Required:    true,
+							ForceNew:    false,
+							Type:        schema.TypeInt,
+							Description: "Proportion of traffic this member is to handle. Must be an integer in the range 1-256",
+						},
+						"min_connections": {
+							Optional:    true,
+							ForceNew:    false,
+							Type:        schema.TypeInt,
+							Description: "Minimum number of concurrent connections a member must always accept",
+						},
+						"max_connections": {
+							Optional: true,
+							ForceNew: false,
+							Type:     schema.TypeInt,
+							Description: "The maximum number of concurrent connections the member can handle. If exceeded" +
```

Comment:
missing a space between the two strings.

Reply:
Thanks. Added.